commands-yml/commands/context/get-context.yml

```diff
@@ -27,8 +27,10 @@ example_usage:
   javascript_wdio:
     |
       let context = driver.context();
-  ruby:
+  ruby_lib:
```

Comment:
What is the intended difference between `ruby_lib` and `ruby` (which is what was there, and remains starting with `commands-yml/commands/device/app/get-app-strings.yml` in this PR)?